docker-compose.yml

```diff
@@ -0,0 +1,21 @@
+version: '3'
+services:
+  ove-apps-latest:
+    image: ove/ove-apps:latest
```

Comment:
Question on versioning instead of `latest`. I'm not an expert on this topic, so wanted to know if it was possible - since this is 100% OSS, we can therefore consider deploying on DockerHub etc.

Reply:
This is a separate issue. I'll sort this out with a custom build script which will take the git tag into account when tagging the docker image.